src/pages/BorrowContent.jsx

```diff
@@ -0,0 +1,580 @@
+                          : null
+          }
+                      </InputDetails>
+                  </InputContainer>
+              </Background>
+              <PrimaryButton positioned={true} text="Supply"/>
+          </SubContainer>
+          <SubContainer>
+              <Background>
+                  <Title>Borrow GARD</Title>
+                  <InputContainer>
+                      <div style={{display: "flex"}}>
+                          <Input
+                          autoComplete="off" 
+                          placeholder={"enter amount"}
+                          type='number'
+                          min="1.00"
+                          step="1"
+                          id="minted"
+                          value={mGARD}
+                          size="small"
+                          onChange={handleBorrowChange}
+                          />
+                          <MaxButton 
+                          onClick={handleMaxBorrow}>
+                              <ToolTip
+                              toolTip={"+MAX"} 
+                              toolTipText={"Click to borrow maximum amount"}
+                              />
+                          </MaxButton>
+                      </div>
+                      <Valuation>$Value: ${mGARD === "" ? 0 : mGARD}</Valuation>
+                      <InputDetails>
+                          {borrowDetails.length && borrowDetails.length > 0 ?
+                          borrowDetails.map((d) => {
+                              return (
+                                  <Item key={d.title}>
+                                      <Effect title={d.title} val={d.val} hasToolTip={d.hasToolTip}></Effect>
+                                  </Item>
+                              )
+                          })
+                          : null
+          }
+                      </InputDetails>
+                  </InputContainer>
+              </Background>
+              <PrimaryButton 
+              positioned={true} 
+              text="Borrow" 
+              disabled={cAlgos == "" || mGARD == ""}
+              onClick={async () => {
+                setLoading(true)
+                try {
+                  const res = await openCDP(
+                    getCollateral(),
+                    getMinted(),
+                    commitChecked,
+                    toWallet,
+                  );
+                  if (res.alert) {
+                    setCreatePositionShown(false)
+                    dispatch(setAlert(res.text));
+                  }
+                } catch (e) {
+                  handleTxError(e, "Error minting CDP");
+                }
+                setLoading(false)
+              }}
+              />
+          </SubContainer>
+      </Container> 
+      <Details className={"borrow"} details={details}/> 
+      </div> :
+      <></> }
+      {cdps == dummyCDPs ? <></> :
+      <div>
+        <Positions/>
+        <PrimaryButton
```

Comment:
This button should be moved to the top of the page, since it activates the dropdown above "your positions". Also, I couldn't figure out how to close the Create Position window. Could you make the clearer? Maybe add a minimize button if there isn't one already? It could even be the same button here that changes its text to "minimize" based on the state of CreatePositionShown...

Reply:
Ok, will do. Also, you probably couldn't figure out how to close it because their wasn't a button - I can add one.